### ISSUE TITLE
AdvancedKea Error

### ISSUE THREAD

**Issue by L34RZ4** [Anything144/Advanced-Kea#3]
![AdvancedKea Error](https://github.com/Anything144/Advanced-Kea/assets/121036226/042fe79e-0ef4-4dd9-a1ff-60162f4fe8ac)

Hello, I am a big fan of your work, I downloaded the advanced kea because the old one is giving me an error. I followed all direction and I received the same unhandled exception error saying too many request,. Please help. Thank you.

**Comment by B1uesh1ft** [comment]
Yep, another of the same issue I reported last October 2023 here: https://github.com/Anything144/Advanced-Kea/issues/1

Seems like everyone is having the same problem with "too many requests".

BTW, love that Webtoon!
